src/renderer/main.js

```diff
@@ -74,24 +74,15 @@ function onState (err, _state) {
   // Lazy-load other stuff, like the AppleTV module, later to keep startup fast
   window.setTimeout(delayedInit, config.DELAYED_INIT)
 
-  // The UI is built with virtual-dom, a minimalist library extracted from React
-  // The concepts--one way data flow, a pure function that renders state to a
-  // virtual DOM tree, and a diff that applies changes in the vdom to the real
-  // DOM, are all the same. Learn more: https://facebook.github.io/react/
-  vdomLoop = mainLoop(state, render, {
-    create: createElement,
-    diff: diff,
-    patch: patch
-  })
-  document.body.appendChild(vdomLoop.target)
-
   // Listen for messages from the main process
   setupIpc()
 
   // Calling update() updates the UI given the current state
   // Do this at least once a second to give every file in every torrentSummary
   // a progress bar and to keep the cursor in sync when playing a video
   setInterval(update, 1000)
+  window.requestAnimationFrame(renderIfNecessary)
+  app = ReactDOM.render(<App state={state} />, document.querySelector('#body'))
```

Comment:
This is for the wrapper `<div>`, needed to avoid React's warning about using `<body>` directly